app/web/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -200,9 +203,11 @@ export default function GroupChatView() {
               <BackIcon />
             </HeaderButton>
 
-            {groupChatUsername ? (
-              <Link to={`/user/${groupChatUsername}`} className={classes.title}>
-                {title || <Skeleton width={100} />}
+            {groupChatUsername && groupChat?.isDm ? (
+              <Link to={`/user/${groupChatUsername}`}>
```

Comment:
```suggestion
              <Link to={`/user/${getDmUsername(groupChatMembersQuery, currentUserId)}`}>
```
I think maybe inline the function call and remove 189-191 might make this slightly clearer too, as we don't really need to call the helper for a group chat

Reply:
P.S. This might even warrant its own component, so the `getDmUsername` stuff can be kept altogether rather than spread across this large component and be used like `<GroupChatTitle />`! But that might be too perfectionist (since you didn't even write most of the code here in the first place!) and so happy to leave it as is so this can be merged soon. Thought I'll still mention that as something for you to think about in the future :)

Reply:
Yes i'll keep that in mind feel free to tag me once this has it's own issue open ;)